x/clp/keeper/calculations.go

```diff
@@ -189,55 +226,110 @@ func CalculatePoolUnits(oldPoolUnits, nativeAssetBalance, externalAssetBalance,
 	return sdk.NewUintFromBigInt(newPoolUnit.RoundInt().BigInt()), sdk.NewUintFromBigInt(stakeUnits.RoundInt().BigInt()), nil
 }
 
-// Add validations for X,x,Y
-//( x^2 * Y ) / ( x + X )^2
-func calcLiquidityFee(X, x, Y sdk.Uint) (sdk.Uint, error) {
-	// if inputs are outside range return error
-
-	//if !ValidateInputs([]sdk.Uint{X, x, Y}) {
-	//	return sdk.ZeroUint(), types.ErrInvalid //error
-	//}
-	////if any input is 0 return 0
-	//if !ValidateZero([]sdk.Uint{X, x, Y}) {
-	//	return sdk.ZeroUint(), nil
-	//}
-	//d := x.Add(X)
-	//denom := d.Mul(d)
-	//return (x.Mul(x).Mul(Y)).Quo(denom), nil
```

Comment:
Sorry, just a pet peeve of mine -- commented code being merged in.... do we need this?

Reply:
+1

Reply:
Fixing

Reply:
Sorry, looks like all that was removed anyway @iverc !